cmd/node/node.go

```diff
@@ -523,9 +523,7 @@ func (app *App) initServices(ctx context.Context,
 		return errors.New("invalid golden atx id")
 	}
 
-	app.keyExtractor = signing.NewPubKeyExtractor(
-		signing.WithVerifierPrefix(app.Config.Genesis.GenesisID().Bytes()),
-	)
+	app.keyExtractor = signing.NewPubKeyExtractor()
```

Comment:
i don't understand this change. in the old code, the default verifier does have the genesis id for prefix

Reply:
I was just trying if this is a possible cause, it is not; the change will be reverted with the next commit. I have a few more spots that could be a cause for issues in systests

Reply:
Undid this change